docs/configuration/integrations/integrations-next/app-o11y-exporter.md

```diff
@@ -0,0 +1,58 @@
++++
+title = "app_o11y_config"
++++
+
+# app_o11y_config
+
+The `app_o11y_config` block configures the `app-o11y-exporter`
+integration, which is a data exporter for logs/metrics/traces/exceptions
+received by the respectful JavaScript agent
+
+Full reference of options:
+
+```yaml
```

Comment:
It seems like the metrics autoscrape/etc fields are missing here

Reply:
a4a5f090